test-app/index.js

```diff
@@ -17,6 +17,7 @@ educandu({
   smtpOptions: 'smtp://localhost:8025/?ignoreTLS=true',
   publicFolders: ['./test-app/dist', './test-app/static'].map(x => path.resolve(x)),
   resources: ['./test-app/resource-overrides.json'].map(x => path.resolve(x)),
+  importApiKey: 'fe160daddb0c44c4963f63ce08272c86',
```

Comment:
I have the feeling this should be renamed to exportAPIKey since this is the key used for a system that imports the data to view what he can import.
Example
elmu is the origin : I would expect this to have an exportAPIKey usable by all importers to get the list of content
oma is the importer: I would expect this to have an importAPIKey to be used when importing from other systems

I know we said we use one API key to rule them all though. I'm just still confused with the naming.

Reply:
True, I renamed it to `exportApiKey`

Reply:
To me it's more confusing to have two names for the exact same key 😄 Buuut anyway

Reply:
Maybe we want to drop the names "import" and "export" completely? I'm also not super happy with the terms. But that would be a discussion out of scope for this PR

Reply:
I think the direction of the exchange needs to be clear. But yeah, out of scope, we can see how working with this feels and if we keep on being confused by the terminology we can revisit and rename anytime anyway